app/controllers/contributors_controller.rb

```diff
@@ -5,7 +5,7 @@ class ContributorsController < ApplicationController
 
   def index
     @contributors = User.with_role(Role::CONTRIBUTOR_ROLES).includes(roles: :team).
-      where("teams.season_id IS NULL OR teams.season_id = ?", Season.current).
+      where("teams.season_id IS NULL OR teams.season_id = ? and teams.kind IS NOT NULL", Season.current).
```

Comment:
I think they are in teams: https://github.com/rails-girls-summer-of-code/rgsoc-teams/blob/master/app/models/team.rb#L115